sdk/identity/azure-identity/src/main/java/com/azure/identity/IntelliJCredentialBuilder.java

```diff
@@ -52,6 +56,35 @@ public IntelliJCredentialBuilder keePassDatabasePath(String databasePath) {
         return this;
     }
 
+    /**
+     * For multi-tenant applications, specifies additional tenants for which the credential may acquire tokens.
```

Comment:
This should also be similar to `AzureCliCredential` and `AzurePowerShellCredential`